Source/Core/VideoBackends/OGL/OGLGfx.h

```diff
@@ -180,7 +105,14 @@ class Renderer : public ::Renderer
   RasterizationState m_current_rasterization_state;
   DepthState m_current_depth_state;
   BlendingState m_current_blend_state;
-  GLuint m_shared_read_framebuffer = 0;
-  GLuint m_shared_draw_framebuffer = 0;
+  u32 m_shared_read_framebuffer = 0;
+  u32 m_shared_draw_framebuffer = 0;
+  float m_backbuffer_scale;
 };
+
+inline OGLGfx* GetOGLGfx()
```

Comment:
Why was this added for OGL?  I did not see it in D3D

Reply:
To remove a bunch of ugly `static_cast<OGLGfx*>(g_gfx.get());` spread throughout the codebase. Though I should probably convert it over to the `GetInstance()` static method style used elsewhere. 

Though in the medium term, both those styles are incompatible with having multiple AbstractGfxs. It can't access a global. 

Might leave this untouched until my async branch